<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpIpServerShowDialog</class>
 <widget class="QDialog" name="TcpIpServerShowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTextBrowser" name="textBrowserServerReceiveArea"/>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBoxServerNextLineShow">
        <property name="text">
         <string>自动换行显示</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxServerSaveToFile">
        <property name="text">
         <string>接收转向文件</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonServerSaveToFile">
        <property name="text">
         <string>保存数据</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonServerClearReceive">
        <property name="text">
         <string>清空接收</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxServerHexShow_Receive">
        <property name="text">
         <string>十六进制显示</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
